datafusion/sqllogictest/test_files/avro.slt

```diff
@@ -124,69 +124,24 @@ STORED AS AVRO
 LOCATION '../../testing/data/avro/simple_fixed.avro';
 
 # test avro query
-query IT
+query error DataFusion error: Arrow error: Schema error: type Utf8View not supported
```

Comment:
🤔  looks like we need to support Utf8View for avro as well

Reply:
Yeah @alamb , i agree, we need to create more sub-tasks.